.github/workflows/pr-triage.yml

```
@@ -10,11 +10,11 @@ env:
 
 jobs:
   content_project:
+    name: did the pr get added to the docs github board? (optional)
```

why is this framed as question? 

```suggestion
    name: pr added to the docs github board (optional)
```